src/YawPitchControl/YawPitchControl.js

```diff
@@ -261,26 +245,27 @@ const YawPitchControl = class YawPitchControl extends Component {
 		}
 	}
 
-	// If new yaw or pitch range is narrow than viewport, it will not apply new ranges.
-	_setYawPitchRange(prevOptions) {
-		const prevYawRange = prevOptions.yawRange;
-		const prevPitchRange = prevOptions.pitchRange;
-		let currYawRange = [];
-		let currPitchRange = [];
-
-		// Restore if it is invalid value
+	_getValidYawRange(newYawRange) {
+		const ratio = YawPitchControl.adjustAspectRatio(this.options.aspectRatio || 1);
 		const fov = this.axes.get().fov;
-		const ratio = YawPitchControl.adjustAspectRatio(this.options.aspectRatio);
 		const horizontalFov = fov * ratio;
+		const isValid = !(newYawRange[1] - newYawRange[0] < horizontalFov);
```

Comment:
Is it right? Isn't your intend following?

```js
// ! is removed
const isValid = (newYawRange[1] - newYawRange[0] < horizontalFov);
```

Although it's your intend I think It would be better to not put ! like below

```js
const isValid = (newYawRange[1] - newYawRange[0] >= horizontalFov);
```

Reply:
Thanks.  I will fix that.